lib/paper_trail/record_trail.rb

```diff
@@ -502,7 +502,7 @@ def habtm_assoc_ids(habtm_assoc)
     end
 
     def log_version_errors(version, action)
-      version.logger.warn(
+      version.logger&.warn(
```

Comment:
Ok, I think I understand now, however according to [this stackoverflow post](http://stackoverflow.com/a/36812667/995874), this functionality was introduced in Ruby 2.3, which would make the gem unusable with earlier Ruby versions, which is not desirable.